assets/sass/alaveteli-org.scss

```diff
@@ -986,3 +986,20 @@ p.ui-dropdown-list-trigger {
 .ui-dropdown-list.active .current-locale:after {
 	border-top-color: #fff;
 }
+
+.fluid-width-video-wrapper {
+    width: 100%;
+    position: relative;
+    padding: 0;
+    padding-top: 75%;
+}
+
+.fluid-width-video-wrapper iframe,
+.fluid-width-video-wrapper object,
+.fluid-width-video-wrapper embed {
+    position: absolute;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, left, position, top, width